ChangeLog

```diff
@@ -84,6 +84,11 @@ Release date: tba
       emitted when item deletion is tried on an object which doesn't
       have this ability. Closes issue #592.
 
+    * Added a new refactoring message, 'consider-merging-isinstance', which is
+      used to check the type of the same object, with different calls, in the
```

Comment:
This is a bit wrong, since it says how the check is implemented, but not what it does.

Maybe ``Added a new refactoring message, ..., which is used to detect isinstance calls, which can be merged together.`` would be better.